composer.json

```diff
@@ -25,22 +25,22 @@
         "php": ">=5.6"
     },
     "require-dev": {
-        "phpcq/all-tasks": "~1.1"
+        "phpcq/all-tasks": "~1.1",
+        "contao/core-bundle": "~4.4"
```

Comment:
I see no usage of ANY Contao class in there... only `Symfony\Component\DependencyInjection\ResettableContainerInterface` (which should be changed to `Symfony\Component\DependencyInjection\ContainerInterface` anyway).

So there is a dependency on `symfony/dependency-injection` but none on `contao/core-bundle`!